github/PullRequest.py

```diff
@@ -383,6 +383,33 @@ def create_issue_comment(self, body):
         )
         return github.IssueComment.IssueComment(self._requester, headers, data, completed=True)
 
+    def create_reviewer_requests(self, *reviewers):
+        """
+        :calls `POST /repos/:owner/:repo/pulls/:number/requested_reviewers`
```

Comment:
Can the link to https://developer.github.com/v3/pulls/review_requests/ be added here?